dataflow/flex-templates/kafka_to_bigquery/README.md

```diff
@@ -200,6 +212,41 @@ gcloud compute instances create-with-container kafka-vm \
   --tags "kafka-server"
 ```
 
+### Sending messages to Kafka server
+
+The Kafka server should be running at this point, but in its current state no
```

Comment:
nit: blending in context with instruction to ssh makes this a little difficult to parse.